.github/workflows/catalyst.yml

```
@@ -0,0 +1,32 @@
+name: 🐧 Catalyst
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.head_ref }}-insituvis
+  cancel-in-progress: true
+
+jobs:
+  catalyst:
+    name: Catalyst
+    runs-on: ubuntu-22.04
+    if: github.event.pull_request.draft == false
+    env:
+      CXX: g++
+      CC: gcc
+    container:
+      image: cwetterernelson/catalyst:v2
+    steps:
+    - uses: actions/checkout@v3
+    - name: Configure
+      run: |
+        cmake -S . -B build          \
```

@c-wetterer-nelson I think your docker container might have a small bug. It sets an env variable `catalyst_DIR`, but I think it needs to be case sensitive:
```suggestion
        export Catalyst_DIR=${catalyst_DIR}
        cmake -S . -B build          \
```
https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure

```
CMake Error at Tools/CMake/AMReXThirdPartyLibraries.cmake:97 (find_package):
  Could not find a package configuration file provided by "Catalyst" with any
  of the following names:

    CatalystConfig.cmake
    catalyst-config.cmake
```

---

An env hint for `Conduit_DIR` also needs to be set :)